engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/interop/generic/GetSourceLocationNode.java

```diff
@@ -24,7 +24,6 @@ public class GetSourceLocationNode extends Node {
   Object execute(Object value) {
     try {
       return EnsoContext.get(this)
-          .getEnvironment()
           .asGuestValue(new EnsoSourceSection(library.getSourceLocation(value)));
```

Comment:
We shouldn't use `asGuestValue` to wrap `EnsoSourceSection`. Rather `EnsoSourceSection` shall implement `EnsoObject` and `@ExportMessage`s directly.